devicetypes/smartthings/zwave-metering-switch.src/zwave-metering-switch.groovy

```diff
@@ -231,10 +231,8 @@ def configure() {
 
 	if (zwaveInfo.mfr == "0086") {	// Aeon Labs meter
 		result << response(encap(zwave.configurationV1.configurationSet(parameterNumber: 80, size: 1, scaledConfigurationValue: 2)))	// basic report cc
-		result << response(encap(zwave.configurationV1.configurationSet(parameterNumber: 101, size: 4, scaledConfigurationValue: 1)))	// report power in watts
+		result << response(encap(zwave.configurationV1.configurationSet(parameterNumber: 101, size: 4, scaledConfigurationValue: 12)))	// report power in watts
```

Comment:
This should be `scaledConfigurationValue: 3` right?

Reply:
@mckeed This is combining 101 with bit value 4 and 102 with bit value 8. Shouldn't this be bit value 12?

Reply:
Yes, it appears this is right, sorry. I was confused because you had the values as 1 and 2 prior to this commit.

Reply:
@mckeed No worries. Thank you for reviewing the PR.